modules/showtime.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-return-assign */
+const showTime = document.querySelector('.showTime');
+
+const currentTime = () => {
+  const dateTime = luxon.DateTime.local();
+  showTime.innerHTML = dateTime.toFormat('MMMM d, yyyy - hh:mm:ss a');
+};
+
```

Comment:
- [ ] Great job for showing the time so far but I think it would be great if you a library [Luxon](https://moment.github.io/luxon/#/install?id=es6) to show the time as required 🚀💯 so please make sure to use luxon and remember to Import it as a module 💯🚀✨

![image](https://user-images.githubusercontent.com/58553711/228111071-8f069eba-0522-4e42-8e91-b402a68150d5.png)